Cargo.toml

```diff
@@ -36,6 +36,7 @@ slog = "2.2"
 slog-envlogger = { version = "2.1.0", optional = true }
 slog-stdlog = { version = "4", optional = true }
 slog-term = { version = "2.4.0", optional = true }
+anyhow = "1.0.32"
```

Comment:
This should be `dev-dependencies` too.

Reply:
> The PR is too large, please add one test (file) at a time.

I will divide them as single group commit , vote and group commit